src/queue.c

```diff
@@ -7886,7 +7895,7 @@ _dispatch_queue_cleanup2(void)
 	// similar non-POSIX API was called
 	// this has to run before the DISPATCH_COCOA_COMPAT below
 	// See dispatch_main for call to _dispatch_sig_thread on linux.
-#ifndef __linux__
+#if 0
 	if (_dispatch_program_is_probably_callback_driven) {
```

Comment:
This seems like an incorrect change here?

Reply:
Yeah, this is definitely incorrect, I didn't notice that was there.